GameWorld/View3D/Services/ResourceLibary.cs

```diff
@@ -58,15 +58,15 @@ public void Initialize(GraphicsDevice graphicsDevice, ContentManager content)
                 CommonSpriteBatch = new SpriteBatch(_graphicsDevice);
 
                 // Load default resources
-                var mr = LoadEffect("Shaders\\Pbr\\MetalRoughness\\MetalRoughness_main", ShaderTypes.Pbs_MetalRough);
+                var mr = LoadEffect("Shaders\\Pbr\\MetalRoughness\\CA_MetalRoughness_main", ShaderTypes.Pbs_MetalRough);
                 var sg = LoadEffect("Shaders\\Pbr\\SpecGloss\\SpecGloss_main", ShaderTypes.Pbr_SpecGloss);
                 LoadEffect("Shaders\\Geometry\\BasicShader", ShaderTypes.BasicEffect);
                 LoadEffect("Shaders\\TexturePreview", ShaderTypes.TexturePreview);
                 LoadEffect("Shaders\\LineShader", ShaderTypes.Line);
                 DefaultFont = _content.Load<SpriteFont>("Fonts//DefaultFont");
 
-                _pbrDiffuse = _content.Load<TextureCube>("textures\\phazer\\DIFFUSE_IRRADIANCE_edited_kloppenheim_06_128x128");
-                _pbrSpecular = _content.Load<TextureCube>("textures\\phazer\\SPECULAR_RADIANCE_edited_kloppenheim_06_512x512");
+                _pbrDiffuse = _content.Load<TextureCube>("textures\\phazer\\SPECULAR_RADIANCE_edited_kloppenheim_06_512x512");
```

Comment:
Specular goes into defuse and the other way around. Is this intentional? SHould not textures be renamed if this is the case?